test/server/cards/02_SHD/leaders/BobaFettDaimyo.spec.ts

```diff
@@ -0,0 +1,141 @@
+describe('Boba Fett, Daimyo', function () {
+    integration(function (contextRef) {
+        describe('Boba Fett\'s leader ability', function () {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['green-squadron-awing', 'cantina-braggart'],
+                        groundArena: ['wilderness-fighter', 'battlefield-marine'],
+                        leader: 'boba-fett#daimyo',
+                        resources: 4,
+                    },
+                    player2: {
+                        hand: ['outer-rim-headhunter'],
+                        groundArena: ['wampa'],
+                    },
+                });
+            });
+
+            it('should give +1/+0 to a friendly unit when play a unit with keyword (first unit play)', function () {
```

Comment:
Great job covering all the keywords! A few more tests it would be good to add - for both the deployed and undeployed side, can you add tests with units which have conditional keywords like Hunting Nexu. Need to confirm that the ability handles both cases correctly when the unit does and does not have the keyword active.

For the deployed ability specifically, could you also test a unit with the Heroic Resolve ability to confirm that (a) gaining keyword from another card works right and (b) the +1 will be correctly applied if the keyword is "for this attack."

Reply:
I skip the undeployed boba with hunting nexu and an aggression unit test, added keyword test is already covered with my Red Three test